tests/acceptance/pageObjects/publicLinkFilesPage.js

```diff
@@ -32,12 +33,31 @@ module.exports = {
         .useStrategy(menuButton)
         .assert.elementNotPresent(menuButton)
         .useCss()
+    },
+    /**
+     * navigates to the root directory of the link share
+     * @param {string} linkName
+     *
+     * @returns {*}
+     */
+    navigateToRootFolder: function (linkName) {
+      if (!linkName) {
+        linkName = 'Public link'
+      }
```

Comment:
you can use it as a default value for the fn arg.

Reply:
Yes but no :smile: . Sometimes the `linkName` will come as `null`, in that case, the default value will not work.